sw/host/opentitanlib/src/transport/chip_whisperer/gpio.rs

```diff
@@ -40,6 +40,7 @@ impl<B: Board> GpioPin for Pin<B> {
 
     fn write(&self, value: bool) -> Result<()> {
         let usb = self.device.borrow();
+        let value = if self.config.invert { !value } else { value };
         usb.pin_set_state(&self.pinname, value)?;
         Ok(())
     }
```

Comment:
Can we add a comment to the `fn write` method in the trait stating that the `value` is sometimes inverted from low/high depending on the pin?